src/microbe_stage/MicrobeAI.cs

```diff
@@ -758,142 +327,57 @@ private void PreyFlee(Random random)
+            microbe.EngulfSize && DistanceFromMe(target.Translation) < 50.0f;
+        targetPosition = target.Translation;
+        microbe.LookAtPoint = targetPosition;
     }
 
     // For doing run and tumble
-    private void DoRunAndTumble(Random random)
+    private void RunAndTumble(Random random)
     {
         // Run and tumble
         // A biased random walk, they turn more if they are picking up less compounds.
+        // The scientifically accurate algorithm has been flipped to account for the compound
+        // deposits being a lot smaller compared to the microbes
         // https://www.mit.edu/~kardar/teaching/projects/chemotaxis(AndreaSchmidt)/home.htm
 
-        var randAngle = previousAngle;
-        float randDist;
+        // If we are still engulfing for some reason, stop
+        microbe.EngulfMode = false;
 
         float compoundDifference = microbe.TotalAbsorbedCompounds.SumValues();
 
-        // Angle should only change if you havent picked up compounds || picked up less compounds
-        if (compoundDifference < 0 && random.Next(0, 10) < 5)
+        // If food density is going down, back up and see if there's some more
+        if (compoundDifference < 0 && random.Next(0, 10) < 9)
         {
-            randAngle = SelectRandomTargetPosition(random);
+            MoveWithRandomTurn(2.5f, 3.0f, random);
         }
 
-        // If last round you had 0, then have a high likelihood of turning
-        if (compoundDifference < Constants.AI_COMPOUND_BIAS && random.Next(0, 10) < 9)
+        // If there isn't any food here, it's a good idea to keep moving
+        if (compoundDifference == 0 && random.Next(0, 10) < 5)
         {
-            randAngle = SelectRandomTargetPosition(random);
+            MoveWithRandomTurn(0.0f, 0.4f, random);
         }
 
-        if (compoundDifference == 0 && random.Next(0, 10) < 9)
+        // If positive last step you gained compounds, so let's stick around
+        if (compoundDifference > 0)
         {
-            randAngle = SelectRandomTargetPosition(random);
-        }
+            // There's a decent chance to turn most of the way around
+            if (random.Next(0, 10) < 4)
+            {
+                MoveWithRandomTurn(0.0f, 3.0f, random);
+            }
 
-        // If positive last step you gained compounds
-        if (compoundDifference > 0 && random.Next(0, 10) < 5)
-        {
-            // If found food subtract from angle randomly;
-            randAngle = previousAngle - random.Next(0.1f, 0.3f);
-            previousAngle = randAngle;
-            randDist = random.Next(200.0f, movementRadius);
-            targetPosition = new Vector3(Mathf.Cos(randAngle) * randDist, 0, Mathf.Sin(randAngle) * randDist);
+            // There's a chance to stop for a bit and letting nutrients soak in
+            // More opportunistic species will do this less.
+            if (random.Next(-Constants.MAX_SPECIES_OPPORTUNISM, Constants.MAX_SPECIES_OPPORTUNISM)
```

Comment:
I see this replaces the plant-like state, my only issue with this is that the old system allowed for there to be low activity high aggression species that just sat there and shot toxins at their enemies and ate their chunks afterwards and only moved after a kill.